plugins/arRestApiPlugin/lib/exception/QubitApiForbiddenException.class.php

```diff
@@ -19,7 +19,6 @@
 
 class QubitApiForbiddenException extends QubitApiException
 {
-    protected $message = 'Forbidden';
```

Comment:
This ends up returning a blank message property (example below triggers this by an attempting to delete the root information object).

```
$ curl -XDELETE -u demo@example.com:demo http://127.0.0.1/api/informationobjects/q4ez-yr97-anxe
   {"id":"forbidden","message":""}
```

Reply:
Good catch @mcantelon, I've restored that line.